js/classes/entities/midground/enemies/yeti.js

```diff
@@ -130,13 +145,24 @@ class Yeti {
                 const rand = Math.floor(Math.random() * 3) + 1;
                 const sfx = SFX["SMASH" + rand];
                 ASSET_MGR.playSFX(sfx.path, sfx.volume);
+
+                // create a wide, horizontal spray of dirt particles
+                const center = new Vector(this.pos.x + Yeti.SCALED_SIZE.x / 2, this.pos.y + Yeti.SCALED_SIZE.y - 10);
+                GAME.addEntity(new ParticleEffect(Vector.subtract(center, new Vector(-100, 0)), ParticleEffect.DIRT_SPRAY));
+                GAME.addEntity(new ParticleEffect(Vector.subtract(center, new Vector(0, 0)), ParticleEffect.DIRT_SPRAY));
```

Comment:
Should it always be dirt? Can we make this dependent upon the terrain or perhaps just change it to snow due to time since yeti will be in snow?